converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -0,0 +1,156 @@
+package gov.cms.qpp.conversion.model.error;
+
+import java.util.Arrays;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
You're blind. There is no import.